deploy-agent/deployd/agent.py

```diff
@@ -120,6 +122,22 @@ def serve_build(self):
         if not self._executor:
             self._executor = Executor(callback=PingServer(self), config=self._config)
         # start to ping server to get the latest deploy goal
+        if len(self._envs) > 0:
+            for status in self._envs.values():
+                # for each container, we check the health status
+                try:
+                    cmd = ['docker', 'inspect', status.report.envName]
```

Comment:
I think we have an util to execute external cmd.

Reply:
probably move it to a helper function in common/utils.py

Reply:
sounds good to me, thanks